Testing/SystemTests/tests/framework/MDNormBackgroundHYSPECTest.py

```diff
@@ -0,0 +1,160 @@
+                     OutputWorkspace='reduced')
+        # Clean memory
+        DeleteWorkspace('split')
+
+        SetUB(Workspace='reduced',
+              a=5.823,
+              b=6.475,
+              c=3.186,
+              u='0,1,0',
+              v='0,0,1')
+        CropWorkspaceForMDNorm(InputWorkspace='reduced',
+                               XMin=-25,
+                               XMax=49,
+                               OutputWorkspace='reduced')
+        ConvertToMD(InputWorkspace='reduced',
+                    QDimensions='Q3D',
+                    Q3DFrames='Q_sample',
+                    OutputWorkspace='md',
+                    MinValues='-11,-11,-11,-25',
+                    MaxValues='11,11,11,49')
+
+        MergeMD(InputWorkspaces='md', OutputWorkspace=merged_md_name)
+
+    @staticmethod
+    def prepare_background(input_md, reference_sample_mde, background_md):
+        dgs_data = CloneWorkspace(input_md)
+        data_MDE = mtd[reference_sample_mde]
+        if mtd.doesExist('background_MDE'):
+            DeleteWorkspace('background_MDE')
+
+        for i in range(data_MDE.getNumExperimentInfo()):
+            phi, chi, omega = data_MDE.getExperimentInfo(i).run().getGoniometer().getEulerAngles('YZY')
+            AddSampleLogMultiple(Workspace=dgs_data,
+                                 LogNames='phi, chi, omega',
+                                 LogValues='{},{},{}'.format(phi,chi,omega))
+            SetGoniometer(Workspace=dgs_data, Goniometers='Universal')
+            ConvertToMD(InputWorkspace=dgs_data,
+                        QDimensions='Q3D',
+                        dEAnalysisMode='Direct',
+                        Q3DFrames="Q_sample",
+                        MinValues='-11,-11,-11,-25',
+                        MaxValues='11,11,11,49',
+                        PreprocDetectorsWS='-',
+                        OverwriteExisting=False,
+                        OutputWorkspace=background_md)
+
+    def requiredMemoryMB(self):
+        return 5000
+
+    def requiredFiles(self):
+        return ['HYS_13656_event.nxs']
+
+    def runTest(self):
+        """ This is the old way to do MDNorm to sample, background and finally clean background from sample
+        Now it serves as a benchmark to generate expected result for enhanced MDNorm
+        """
+        # Set facility that load data
+        config.setFacility('SNS')
+        Load(Filename='HYS_13656', OutputWorkspace='sum')
+        SetGoniometer(Workspace='sum', Axis0='s1,0,1,0,1')
+
+        # prepare sample MD
+        self.prepare_md(input_ws_name='sum', merged_md_name='merged', min_log_value=10, max_log_value=12)
+        # Prepare background workspace
+        # old way - use reduced_1 as the background
+        self.prepare_background(input_md='reduced_1', reference_sample_mde='merged',
+                                background_md='background_MDE')
+
+        # do MDNorm to sample data
+        MDNorm(InputWorkspace='merged',
```

Comment:
@jrmcgaha-dev  Could you run @AndreiSavici 's script to generate output and compare with this?  The modification to Andrie's shall be 
1. Load only run 13655
2. Minimum log value in GenerateEventFilters shall be set to 10, which is not specified in his script
3. Maximum log value in GenerateEventFilters shall be set to 12 from 90.

Reply:
1. Correction: 13656